### ISSUE TITLE
Globally shared labels

### ISSUE THREAD

**Issue by erlend-sh** [omnivore-app/omnivore#1842]
Labels could be collectively curated by letting users opt in to have their labels be shared globally. I’d like to know what labels have already been added by other netizens to a popular article. Then I can agree with the labels (signaling label accuracy) by adopting those labels as my own.

This would also enable browsing of public labels and the articles therein, e.g. the thousands of articles within #opensource, or some dozens within #opensource+burnout.

**Comment by jacksonh** [comment]
Yeah interesting idea, I wonder how we could do this well without leaking any private data.

**Comment by Podginator** [comment]
Closing due to age, and the self-hosted nature of Omnivore. Done as a batch action. If this issue still exists, please reopen it.
